.gitignore

```diff
@@ -39,3 +39,5 @@ website/vendor
 *.winfile eol=crlf
 
 terraform-provider-astra
+
+docs/

```

Comment:
Why are the docs removed?  Aren't they needed here to be uploaded to the site?  I think it's also nice to have them checked in so it's easy to see changes in PRs.

Reply:
Having this as a part of the branch is a bit misleading because it doesn't always reflect the actual state. I bet that folks committing changes are not running `make docs` every time a change is made in code. So you need to pull and make if you want to see actual state.

Please correct me if I missed something happening.

Reply:
The CI/docs job checks that any doc changes have been generated and committed.  It's failing now that git is no longer ignoring that directory.

You might be right about goreleaser regenerating the docs during upload, but I think it's still helpful to be able to view the doc changes directly in PRs.  Since they are markdown it's easy to view them on the branch.  Also, the example terraform providers from Hashicorp commit the generated docs into the repo, for example see this project (https://github.com/hashicorp/terraform-provider-scaffolding-framework)